File: app/src/main/java/com/bovink/androidlearning/AppDatabase.java
package com.bovink.androidlearning;

import androidx.room.Database;
import androidx.room.RoomDatabase;

/**
 * @author bovink
 * @since 2021/1/12
 */
@Database(entities = {User.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract UserDao userDao();

}

File: app/src/main/java/com/bovink/androidlearning/AppDatabase.kt
package com.bovink.androidlearning

import androidx.room.Database
import androidx.room.RoomDatabase

/**
 * @author bovink
 * @since 2021/6/9
 */
@Database(entities = arrayOf(User::class), version = 1, exportSchema = false)
abstract class AppDatabase : RoomDatabase() {

    companion object {
        val dabase_name = "Persons"
    }

    abstract fun userDao(): UserDao
}